### ISSUE TITLE
SyntaxError: Unexpected token \ in compiler.js

### ISSUE THREAD

**Issue by lvarayut** [jantimon/favicons-webpack-plugin#3]
I got an error while running webpack. The cause of error seems to be `JSON.parse(childCompilation.assets[outputName].source())` in the [compiler.js](https://github.com/jantimon/favicons-webpack-plugin/blob/master/lib/compiler.js#L83). 

**webpack.config.js**:

```js
    var FaviconsWebpackPlugin = require('favicons-webpack-plugin');
    ...
    plugins: [
      ...
      new FaviconsWebpackPlugin('./client/assets/favicon.png')
    ]
```

**Log**:

```bash
/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/webpack-core/lib/NormalModuleMixin.js:151
                                throw e;
                                ^

SyntaxError: Unexpected token \
    at Object.parse (native)
    at /Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/favicons-webpack-plugin/lib/compiler.js:83:23
    at Compiler.<anonymous> (/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/webpack/lib/Compiler.js:214:10)
    at /Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/webpack/lib/Compiler.js:403:12
    at Compiler.next (/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/tapable/lib/Tapable.js:67:11)
    at Compiler.<anonymous> (/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/webpack/lib/CachePlugin.js:40:4)
    at Compiler.applyPluginsAsync (/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/tapable/lib/Tapable.js:71:13)
    at Compiler.<anonymous> (/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/webpack/lib/Compiler.js:400:9)
    at Compilation.<anonymous> (/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/webpack/lib/Compilation.js:577:13)
    at Compilation.applyPluginsAsync (/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/tapable/lib/Tapable.js:60:69)
    at Compilation.<anonymous> (/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/webpack/lib/Compilation.js:572:10)
    at Compilation.next (/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/tapable/lib/Tapable.js:67:11)
    at Compilation.<anonymous> (/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/favicons-webpack-plugin/lib/compiler.js:57:7)
    at Compilation.applyPluginsAsync (/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/tapable/lib/Tapable.js:71:13)
    at Compilation.<anonymous> (/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/webpack/lib/Compilation.js:567:9)
    at Compilation.applyPluginsAsync (/Users/lvarayut/Documents/Web development/Projects/relay-fullstack/node_modules/tapable/lib/Tapable.js:60:69)
```

**Comment by jantimon** [comment]
Hmm could you please add a console log and add the content of `(childCompilation.assets[outputName].source())` to this issue?

**Comment by lvarayut** [comment]
Yes, I'll do it this evening. Will get back to you.

**Comment by leMaik** [comment]
Same here, see the content of my `childCompilation.assets[outputName].source()` below. Looks like everything is escaped where it shouldn't be.

```json
{\"outputFilePrefix\":\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/\",\"html\":[\"<link rel=\\\"icon\\\" type=\\\"image/png\\\" sizes=\\\"192x192\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/android-chrome-192x192.png\\\">\",\"<meta name=\\\"mobile-web-app-capable\\\" content=\\\"yes\\\">\",\"<meta name=\\\"theme-color\\\" content=\\\"#fff\\\">\",\"<meta name=\\\"application-name\\\" content=\\\"Citclick\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"57x57\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-57x57.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"60x60\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-60x60.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"72x72\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-72x72.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"76x76\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-76x76.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"114x114\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-114x114.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"120x120\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-120x120.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"144x144\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-144x144.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"152x152\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-152x152.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"180x180\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-180x180.png\\\">\",\"<meta name=\\\"apple-mobile-web-app-capable\\\" content=\\\"yes\\\">\",\"<meta name=\\\"apple-mobile-web-app-status-bar-style\\\" content=\\\"black-translucent\\\">\",\"<meta name=\\\"apple-mobile-web-app-title\\\" content=\\\"Citclick\\\">\",\"<link rel=\\\"icon\\\" type=\\\"image/png\\\" sizes=\\\"16x16\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/favicon-16x16.png\\\">\",\"<link rel=\\\"icon\\\" type=\\\"image/png\\\" sizes=\\\"32x32\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/favicon-32x32.png\\\">\",\"<link rel=\\\"icon\\\" type=\\\"image/png\\\" sizes=\\\"96x96\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/favicon-96x96.png\\\">\",\"<link rel=\\\"icon\\\" type=\\\"image/png\\\" sizes=\\\"230x230\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/favicon-230x230.png\\\">\",\"<link rel=\\\"shortcut icon\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/favicon.ico\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 320px) and (device-height: 480px) and (-webkit-device-pixel-ratio: 1)\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-320x460.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 320px) and (device-height: 480px) and (-webkit-device-pixel-ratio: 2)\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-640x920.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 320px) and (device-height: 568px) and (-webkit-device-pixel-ratio: 2)\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-640x1096.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 375px) and (device-height: 667px) and (-webkit-device-pixel-ratio: 2)\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-750x1294.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 414px) and (device-height: 736px) and (orientation: landscape) and (-webkit-device-pixel-ratio: 3)\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-1182x2208.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 414px) and (device-height: 736px) and (orientation: portrait) and (-webkit-device-pixel-ratio: 3)\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-1242x2148.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 768px) and (device-height: 1024px) and (orientation: landscape) and (-webkit-device-pixel-ratio: 1)\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-748x1024.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 768px) and (device-height: 1024px) and (orientation: portrait) and (-webkit-device-pixel-ratio: 1)\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-768x1004.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 768px) and (device-height: 1024px) and (orientation: landscape) and (-webkit-device-pixel-ratio: 2)\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-1496x2048.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 768px) and (device-height: 1024px) and (orientation: portrait) and (-webkit-device-pixel-ratio: 2)\\\" href=\\\"/icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-1536x2008.png\\\">\"],\"files\":[\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/android-chrome-36x36.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/android-chrome-48x48.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/android-chrome-72x72.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/android-chrome-96x96.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/android-chrome-144x144.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/android-chrome-192x192.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-57x57.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-60x60.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-72x72.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-76x76.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-114x114.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-120x120.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-144x144.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-180x180.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-152x152.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-icon-precomposed.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/favicon-16x16.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/favicon-32x32.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/favicon-96x96.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/favicon.ico\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/favicon-230x230.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-320x460.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-640x920.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-640x1096.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-748x1024.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-768x1004.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-750x1294.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-1242x2148.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-1182x2208.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-1496x2048.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/apple-touch-startup-image-1536x2008.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/firefox_app_60x60.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/firefox_app_128x128.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/firefox_app_512x512.png\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/manifest.json\",\"icons-f59f43b9a28f9cd6d8b1143fbb7c2ce2/manifest.webapp\"]}
```

**Comment by lvarayut** [comment]
I got the exact same result as @leMaik. It seems like the result of `childCompilation.assets[outputName].source()` has been escaped many time.

**Comment by jantimon** [comment]
Does this also happen if you disable the disk cache?

**Comment by lvarayut** [comment]
Yes, I got the same result after setting `persistentCache: false`.

**Comment by jantimon** [comment]
Hmm really sorry for this.. actually I don't understand why this isn't happening in the tests..
Do you have any idea how this might happen?

**Comment by jantimon** [comment]
Could you please also add a console.log for this row:

https://github.com/jantimon/favicons-webpack-plugin/blob/master/lib/loader.js#L37

```js
console.log( '// LOADER START //' + JSON.stringify(iconResult) + '// LOADER END //');
```

Maybe it's already wrong here.

**Comment by lvarayut** [comment]
Actually, I'm looking at that line also. I think that the error might start from that line. Here is the result:

```
// LOADER START //"{\"outputFilePrefix\":\"icons-a5a719ae21eecc774beecb32f0d38562/\",\"html\":[\"<link rel=\\\"icon\\\" type=\\\"image/png\\\" sizes=\\\"192x192\\\" href=[121/17051]
5a719ae21eecc774beecb32f0d38562/android-chrome-192x192.png\\\">\",\"<meta name=\\\"mobile-web-app-capable\\\" content=\\\"yes\\\">\",\"<meta name=\\\"theme-color\\\" content=\\\"#ff
f\\\">\",\"<meta name=\\\"application-name\\\" content=\\\"relay-fullstack\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"57x57\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d
38562/apple-touch-icon-57x57.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"60x60\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-60x60.png\\\">\",\"
<link rel=\\\"apple-touch-icon\\\" sizes=\\\"72x72\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-72x72.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\
\\"76x76\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-76x76.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"114x114\\\" href=\\\"icons-a5a719ae21ee
cc774beecb32f0d38562/apple-touch-icon-114x114.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"120x120\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-
120x120.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"144x144\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-144x144.png\\\">\",\"<link rel=\\\"app
le-touch-icon\\\" sizes=\\\"152x152\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-152x152.png\\\">\",\"<link rel=\\\"apple-touch-icon\\\" sizes=\\\"180x180\\\
" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-180x180.png\\\">\",\"<meta name=\\\"apple-mobile-web-app-capable\\\" content=\\\"yes\\\">\",\"<meta name=\\\"apple
-mobile-web-app-status-bar-style\\\" content=\\\"black-translucent\\\">\",\"<meta name=\\\"apple-mobile-web-app-title\\\" content=\\\"relay-fullstack\\\">\",\"<link rel=\\\"icon\\\"
 type=\\\"image/png\\\" sizes=\\\"16x16\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/favicon-16x16.png\\\">\",\"<link rel=\\\"icon\\\" type=\\\"image/png\\\" sizes=\\\"32x32\
\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/favicon-32x32.png\\\">\",\"<link rel=\\\"icon\\\" type=\\\"image/png\\\" sizes=\\\"96x96\\\" href=\\\"icons-a5a719ae21eecc774beec
b32f0d38562/favicon-96x96.png\\\">\",\"<link rel=\\\"icon\\\" type=\\\"image/png\\\" sizes=\\\"230x230\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/favicon-230x230.png\\\">\"
,\"<link rel=\\\"shortcut icon\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/favicon.ico\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 320px) a
nd (device-height: 480px) and (-webkit-device-pixel-ratio: 1)\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-startup-image-320x460.png\\\">\",\"<link rel=\\\"apple-
touch-startup-image\\\" media=\\\"(device-width: 320px) and (device-height: 480px) and (-webkit-device-pixel-ratio: 2)\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touc
h-startup-image-640x920.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 320px) and (device-height: 568px) and (-webkit-device-pixel-ratio: 2)\\\" h
ref=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-startup-image-640x1096.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 375px) and (devic
e-height: 667px) and (-webkit-device-pixel-ratio: 2)\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-startup-image-750x1294.png\\\">\",\"<link rel=\\\"apple-touch-st
artup-image\\\" media=\\\"(device-width: 414px) and (device-height: 736px) and (orientation: landscape) and (-webkit-device-pixel-ratio: 3)\\\" href=\\\"icons-a5a719ae21eecc774beecb
32f0d38562/apple-touch-startup-image-1182x2208.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 414px) and (device-height: 736px) and (orientation:
portrait) and (-webkit-device-pixel-ratio: 3)\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-startup-image-1242x2148.png\\\">\",\"<link rel=\\\"apple-touch-startup-
image\\\" media=\\\"(device-width: 768px) and (device-height: 1024px) and (orientation: landscape) and (-webkit-device-pixel-ratio: 1)\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d
38562/apple-touch-startup-image-748x1024.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 768px) and (device-height: 1024px) and (orientation: portr
ait) and (-webkit-device-pixel-ratio: 1)\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-startup-image-768x1004.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\
\\" media=\\\"(device-width: 768px) and (device-height: 1024px) and (orientation: landscape) and (-webkit-device-pixel-ratio: 2)\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/
apple-touch-startup-image-1496x2048.png\\\">\",\"<link rel=\\\"apple-touch-startup-image\\\" media=\\\"(device-width: 768px) and (device-height: 1024px) and (orientation: portrait)
and (-webkit-device-pixel-ratio: 2)\\\" href=\\\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-startup-image-1536x2008.png\\\">\"],\"files\":[\"icons-a5a719ae21eecc774beecb32f0
d38562/android-chrome-48x48.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/android-chrome-192x192.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/android-chrome-72x72.png\",\"icons-a
5a719ae21eecc774beecb32f0d38562/android-chrome-36x36.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/android-chrome-96x96.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/android-chrom
e-144x144.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-57x57.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-60x60.png\",\"icons-a5a719ae21eecc774
beecb32f0d38562/apple-touch-icon-72x72.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-114x114.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-76x76.
png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-152x152.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-120x120.png\",\"icons-a5a719ae21eecc774beecb3
2f0d38562/apple-touch-icon-180x180.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-144x144.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon.png\",\"ic
ons-a5a719ae21eecc774beecb32f0d38562/apple-touch-icon-precomposed.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/favicon-16x16.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/favicon
-96x96.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/favicon-32x32.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/favicon.ico\",\"icons-a5a719ae21eecc774beecb32f0d38562/favicon-230
x230.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-startup-image-320x460.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-startup-image-640x1096.png\",\"icons
-a5a719ae21eecc774beecb32f0d38562/apple-touch-startup-image-640x920.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-startup-image-748x1024.png\",\"icons-a5a719ae21eecc774
beecb32f0d38562/apple-touch-startup-image-768x1004.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-startup-image-750x1294.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/a
pple-touch-startup-image-1182x2208.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-startup-image-1242x2148.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-star
tup-image-1496x2048.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/apple-touch-startup-image-1536x2008.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/firefox_app_128x128.png\",\"ico
ns-a5a719ae21eecc774beecb32f0d38562/firefox_app_60x60.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/firefox_app_512x512.png\",\"icons-a5a719ae21eecc774beecb32f0d38562/manifest.json
\",\"icons-a5a719ae21eecc774beecb32f0d38562/manifest.webapp\"]}"// LOADER END //
```

**Comment by leMaik** [comment]
I just found out that if I remove `devtool: 'eval'` from my webpack.conf.js, the problem is resolved.

**Comment by jantimon** [comment]
wow awesome! thank you so much - I'll run an eval as I used in the html-webpack-plugin.. thought I would get around this ;)

**Comment by jantimon** [comment]
will be fixed by tomorrow

**Comment by leMaik** [comment]
Awesome! :) Thank you very much.

**Comment by jantimon** [comment]
Fixed in 0.0.6 added devtool: eval to the tests
